push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/PushNotificationSender.java

```diff
@@ -27,6 +27,8 @@
  */
 public interface PushNotificationSender {
 
+    String AEROGEAR_PUSH_ID = "aerogear-push-id";
```

Comment:
add it on the message class?

On Wednesday, April 29, 2015, Sebastien Blanc <notifications@github.com>
wrote:

> In
> push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/PushNotificationSender.java
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/540#discussion_r29352246>
> :
>
> > @@ -27,6 +27,8 @@
> >   */
> >  public interface PushNotificationSender {
> >
> > +    String AEROGEAR_PUSH_ID = "aerogear-push-id";
>
> I understand but it's not only the header name, it's also a key of the
> payload.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/540/files#r29352246>
> .
>


-- 
Sent from Gmail Mobile